ui/src/main/webapp/src/app/components/reports/migration-issues/migration-issues-table.component.ts

```diff
@@ -79,4 +83,13 @@ export class MigrationIssuesTableComponent implements OnInit {
 
         return this.problemSummariesFiles.get(issue);
     }
+
+    navigateToSource(file:any) {
+        let fileModel = <FileModel>new GraphJSONToModelService().fromJSON(file, this._http, FileModel);
```

Comment:
This doesn't generally make http requests, although the objects that it creates can potentially make them while crawling the graph.